SourceryRuntime/Sources/TypeName.swift

```diff
@@ -66,23 +66,24 @@ public final class TypeName: NSObject, AutoCoding, AutoEquatable, AutoDiffable,
             self.isOptional = false
         } else {
             name = name.bracketsBalancing()
+            let trimmedInoutName = name.trimmingPrefix("inout ")
```

Comment:
I think it's better to trim whitespaces using character set. SourceKit only trims spaces around type name, which is `inout` is part of, so it can be any number of spaces in between.